zetaclient/chains/evm/observer/inbound.go

```diff
@@ -446,13 +446,13 @@ func (ob *Observer) ObserverTSSReceive(ctx context.Context, startBlock, toBlock
 	// query incoming gas asset
 	for bn := startBlock; bn <= toBlock; bn++ {
 		// observe TSS received gas token in block 'bn'
-		err := ob.ObserveTSSReceiveInBlock(ctx, bn)
+		err := ob.ObserveTSSReceiveInBlockAndOutbound(ctx, bn)
```

Comment:
maybe we could have the comment updated as `// observe TSS received gas token and outbounds txns signed by TSS in block 'bn'` ?